src/api/models.py

```diff
@@ -119,6 +119,7 @@ class Product(db.Model):
    id = db.Column(db.Integer,autoincrement=True, primary_key=True)
    product_name=db.Column(db.String, nullable=False)
    text=db.Column(db.String, nullable=False)
+   shopname=db.Column(db.String, nullable=False)
```

Comment:
No tengo claro que necesitéis un shopname, yo lo planteé al principio y me dijisteis que no

Reply:
Si lo he hablado con el grupo y es necesario como me han dicho